tests/v2/validation/provisioning/rke1/provisioning_node_driver_test.go

```diff
@@ -217,11 +220,19 @@ func (r *RKE1NodeDriverProvisioningTestSuite) testProvisioningRKE1Cluster(client
 	require.NoError(r.T(), err)
 	assert.Equal(r.T(), clusterName, clusterResp.Name)
 	assert.Equal(r.T(), nodePoolName, nodePool.Name)
+	assert.Equal(r.T(), kubeVersion, clusterResp.RancherKubernetesEngineConfig.Version)
+
+	err = nodestat.IsNodeReady(client, clusterResp.ID)
+	require.NoError(r.T(), err)
 
 	clusterToken, err := clusters.CheckServiceAccountTokenSecret(client, clusterName)
 	require.NoError(r.T(), err)
 	assert.NotEmpty(r.T(), clusterToken)
 
+	podResults, podErrors := pods.StatusPods(client, clusterResp.ID)
+	assert.NotEmpty(r.T(), podResults)
+	assert.Empty(r.T(), podErrors)
+
 	return clusterResp, nil
 }
 
```

Comment:
I like that idea, we can do that with the AKS scaling test as well as you mentioned below.

Reply:
Agreed, also another point to consider is while running the tests with subtest full name regex f.e. `TestRKE1ProvisioningTestSuite/TestProvisioningRKE1ClusterDynamicInput/Admin_User_Node_Provider:_aws_Kubernetes_version:_v1.24.9-rancher1-1_cni:_calico` this also includes `Admin_User_Node_Provider:_aws_Kubernetes_version:_v1.24.9-rancher1-1_cni:_calico_scaling` since regex accepts this as _contains_. 

So I would say we need either a way of not using exactly the same subtest string, or separating scale tests if that's not possible.

Reply:
I think I have this addressed, I'm going to push a commit soon with the changes.